src/test/scala/com/mowczare/kafka/streams/example/stream/FrequencyStreamTest.scala

```diff
@@ -0,0 +1,57 @@
+package com.mowczare.kafka.streams.example.stream
+
+import com.madewithtea.mockedstreams.MockedStreams
+import com.mowczare.kafka.streams.example.model.InputEvent
+import com.mowczare.kafka.streams.example.serde.SerdeUtil
+import com.mowczare.kafka.streams.pds.frequency.ItemSketchWrap
+import com.yahoo.sketches.frequencies.ErrorType
+import org.apache.kafka.streams.scala.Serdes
+import org.scalatest.{FunSuite, Matchers}
+
+class FrequencyStreamTest extends FunSuite with Matchers {
+
+  val admissibleError = 0.1
```

Comment:
You may consider moving the test-spec values to common trait.

Reply:
Also, may be the `admissibleError` computed somehow based on Hll/Sketch initial configuration?

Reply:
I don't know if admissibleError should be computed based on initial configuration. 
Testing probabilistic data structures accuracy is not trivial. Unit test are not suitable for testing accuracy, because unit tests rely on 0-1 logic, while testing accuracy should rely on fuzzy logic.

I wrote these tests with following idea: 
Ok, I should somehow test stream which uses sketches. I assume, that theses sketches are correct, so I don't want to test them, but proper usage of them. 
One of the simplest way to do so, is to check, if final result is more or less accurate. In my world, more or less is 10% :)